### PR TITLE
Adds tests for initial data returned by the monitor functions

### PR DESCRIPTION
**Pull request by roytman**
Adds tests for initial data returned by the monitor functions

### CODE REVIEW

File: pkg/ovsdb/monitorining_test.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package ovsdb
```
Comment (hunchback) [inline pkg/ovsdb/monitorining_test.go:1]: you may want to consider renaming file to `monitoring_test.go`

File: pkg/ovsdb/service_test.go [hunk 2]
```diff
@@ -5,23 +5,25 @@ import (
...
 
-	"github.com/ibm/ovsdb-etcd/pkg/common"
-
 	"github.com/stretchr/testify/assert"
+
+	"github.com/ibm/ovsdb-etcd/pkg/common"
 )
 
-const baseTransact string = "../../tests/data/transact/"
+const BaseData string = "../../tests/data/"
```
Comment (hunchback) [inline pkg/ovsdb/service_test.go:13]: this is only going to be used internally ... anyone using `BaseData` from a different location will fail

File: pkg/ovsdb/service_test.go [hunk 3]
```diff
@@ -5,23 +5,25 @@ import (
...
+const BaseTransact string = BaseData + "/transact/"
```
Comment (hunchback) [inline pkg/ovsdb/service_test.go:14]: same as above

File: pkg/ovsdb/monitorining_test.go (hunk 1 above)
Comment (roytman) [reply pkg/ovsdb/monitorining_test.go:1]: done

File: pkg/ovsdb/service_test.go (hunk 2 above)
Comment (roytman) [reply pkg/ovsdb/service_test.go:13]: ok

File: pkg/ovsdb/service_test.go (hunk 3 above)
Comment (roytman) [reply pkg/ovsdb/service_test.go:14]: ok

**Pull request merged**